test/helpers/increaseTime.js

```diff
@@ -29,7 +29,7 @@ export default function increaseTime(duration) {
 }
 
 // sets time to given timestamp based on current block time
-export function setTimeTo(timestamp) {
+export async function setTimeTo(timestamp) {
```

Comment:
Yeah, my main problem is the inconsistency with `increaseTime` before it. That one should also have an `async` flag then.

Or, alternatively, replace `sendAsync` with `send` and make both function synchronous. This is how it is done elswhere (specifically in latestTimestamp). Again, for consistency we should do the same everywhere.